core/src/main/java/com/rim/IceField/Game.java

```diff
@@ -3,28 +3,33 @@
 import java.util.ArrayList;
 
 public class Game {
-    private Map map;
+    private static Map map;
 
     public Game(){
-        this.map = new Map();
+        map = new Map();
     }
 
     public Map getMap() {
         return map;
     }
 
-    public void setMap(Map map) {
-        this.map = map;
-    }
 
     public static void GameOver()
     {
-        System.out.println("Game Over!");
+
+        for (Iceberg ice: map.getIcebergs() ){
+            if ( (ice.getCurrentPlayers().size() == ice.getNum())  &&  (Inventory.isFlareGunAssembled()) )
+                System.out.println("Game Over!");
```

Comment:
You can also print that, in this case, its a win and if someone dies than it would be a lost game